AWS Lambda initial POC (#139)
run-house/runhouse: docs/api/python/lambda_function.rst

> Lambda Function

Reviewer: Is it worth including a setup section (e.g. what roles / permissions to configure with AWS), similar to SageMaker (https://www.run.house/docs/api/python/cluster#sagemaker-hardware-setup) or On-demand (https://www.run.house/docs/api/python/cluster#ondemandcluster-hardware-setup)?

====================================

.. note::

    Lambda Function support is an alpha and under active development.
    Please report any bugs or let us know of any feature requests.

> An AWS Lambda Function is a serverless provided by AWS. It allows to run applications and backend services without

Reviewer: In this section would be worth highlighting the two core usage paths - passing in a function callable, or the typical lambda flow where you provide paths to code and a handler function name

> An AWS Lambda Function is a serverless provided by AWS. It allows to run applications and backend services without

Reviewer: ```suggestion
A `Lambda Function <https://aws.amazon.com/lambda/>`_ is a serverless compute service by AWS. It allows you to run applications and backend services without
```
